server/app.py

```diff
@@ -11,8 +11,7 @@ def index():
 
 @app.route('/heatmap', methods=["POST"])
 def heatmap():
-    # Ryan, this is where you would plug in your function to generate the heatmap
-    features = { 'batter' : 545361 }
+    features = { 'batter' : request.form['batter'] }
```

Comment:
change to:
features = { 'batter' : int(request.form['batter']) }

Back end code doesn't break if 'batter' is a string, just treats it as out-of-domain, which I guess means the heatmap represents some kind of average heatmap over all players.